sql/moz-fx-data-shared-prod/activity_stream/dataset_metadata.yaml

```diff
@@ -0,0 +1,11 @@
+friendly_name: Activity Stream
```

Comment:
We fill out the three activity stream datasets as an example here. The `_bi` dataset is a "custom" one with an additional ACL entry. The `_derived` directory didn't previously exist, but we add it here for completeness since the pipeline currently automatically creates a `_derived` dataset per document namespace.